Corona-Warn-App/src/main/res/navigation/srs_nav_graph.xml

```diff
@@ -34,12 +34,30 @@
         android:id="@+id/srsSubmissionTypeSelectionFragment"
         android:name="de.rki.coronawarnapp.srs.ui.typeselection.SrsTypeSelectionFragment"
         android:label="SrsSubmissionTypeSelectionFragment"
-        tools:layout="@layout/fragment_srs_type_selection" />
+        tools:layout="@layout/fragment_srs_type_selection">
+
+        <action
+            android:id="@+id/action_srsSubmissionTypeSelectionFragment_to_mainFragment"
+            app:destination="@id/status_nav_graph"
+            app:popUpTo="@id/nav_graph"
+            app:popUpToInclusive="true" />
+
+        <action
+            android:id="@+id/action_srsSubmissionTypeSelectionFragment_to_srsCheckinsFragment"
+            app:destination="@id/srsCheckinsFragment" />
+
+        <deepLink app:uri="cwa://srs-consent/openTypeSelection" />
+    </fragment>
     <fragment
         android:id="@+id/srsCheckinsFragment"
         android:name="de.rki.coronawarnapp.srs.ui.checkins.SrsSubmissionCheckinsFragment"
         android:label="SrsCheckinsFragment"
-        tools:layout="@layout/fragment_srs_type_selection" />
+        tools:layout="@layout/fragment_srs_type_selection">
+        <argument
+            android:name="openTypeSelection"
```

Comment:
We need to pas this argument to the symptoms fragment as well.

Reply:
Also, I would call this argument a different name. We also have `openTypeSelection` for the consent screen and that is a boolean. Here it's the actual `SrsSubmissionType`. Maybe it's better if I add the arguments in my symptom intro flow PR as you can't use them now anyway and we would have merge conflicts if you added them too.

Reply:
Really? This was just introduced when we did not know which test was used. After this fragment we know, so we don't need that flag anymore. For naming sync with @mtwalli, it was his suggestion

Reply:
@schauersbergern  is right, here we don't need the type , He we actually should select the type. BUT I don't think that we need this flag either. 

What needs `type` parameter is either `checkins` or `symptoms` screen

Reply:
I removed the arguments for now